recipes/poolsnp/build.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+set -ex
+
+BASE_DIR=$(dirname "$0")
+SCRIPTS_DIR="$BASE_DIR/scripts"
+
+if [ ! -d "$SCRIPTS_DIR" ]; then
+    echo "The scripts directory is missing. It should be placed in the same directory as the PoolSNP.sh script."
+    exit 1
+fi
+
+chmod +x PoolSNP.sh
+
```

Comment:
```suggestion
mkdir -p ${PREFIX}/bin
```

Reply:
Thanks for the suggestion, I followed the style to create all necessary directories